### PR TITLE
test: test

### PR DESCRIPTION
**Pull request by marusak**
I am having hard time reproducing issues that CI has locally, thus I'll try this PR.
Starting simple.

### CODE REVIEW

File: test/check-application [hunk 1]
```diff
@@ -1322,7 +1322,7 @@ class TestApplication(testlib.MachineCase):
 
         b.click("a:contains('Logs')")
         b.wait_text(".container-logs .xterm-accessibility-tree > div:nth-child(1)", "1")
-        b.wait_text(".container-logs .xterm-accessibility-tree > div:nth-child(6)", "6")
+        b.wait_in_text(".container-logs .xterm-accessibility-tree", "6")
```
Comment (martinpitt) [inline test/check-application:1325]: Any chance the test could log something a little longer/specific than "6"?

**Review by martinpitt — APPROVED** [review]
Danke! Tests are still running, but these two look okay (not *thrilled* about the timeouts, but :man_shrugging:

File: test/check-application (hunk 1 above)
Comment (marusak) [reply test/check-application:1325]: they surely could. But they log numbers from one onwards. And tests check that we see them from 1 but also that they are being followed. So I could change it for example not to start from one, but from 1000 and then check for 1006 - if that would be better?

Comment (martinpitt) [reply test/check-application:1325]: I thought about something like `containerNameLog%i`. Either way, fine for a follow-up.

**Pull request merged**